src/backend/distributed/sql/udfs/create_time_partitions/latest.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE FUNCTION pg_catalog.create_time_partitions(
+    table_name regclass,
+    partition_interval INTERVAL,
+    to_value timestamptz,
+    from_value timestamptz DEFAULT now())
```

Comment:
Optional: 
I think after playing with these functions for a while, I feel like start_from and end_at are slightly better names after all, because there are 2 semantic differences between these arguments and the FROM/TO in the partitioning syntax.
1) They refer to the entire range of partitions
2) They do not necessarily refer to the exact start/end of a partition

Plus there is a cosmetic difference: FROM always comes before TO in the partitioning syntax.

If I think of this function as a sentence it would be "create time partitions for table events with time interval '1 month', end at '2021-12-31', and start from '2021-01-01'"